starboard/linux/x64x11/shared/platform_configuration/BUILD.gn

```diff
@@ -30,6 +30,12 @@ config("platform_configuration") {
 
 config("libraries") {
   configs = [ "//starboard/linux/shared/platform_configuration:libraries" ]
+  if (enable_iamf_decode) {
+    libs = [
+      "//third_party/libiamf/platforms/linux/libiamf.a",
+      "//third_party/libiamf/source/code/dep_codecs/lib/libfdk-aac.a",
```

Comment:
Is libfdk-aac.a needed for iamf decoder?

Reply:
It's needed for linux as the generated `libiamf.a` requires it. It's possible to generate `libiamf.a` without depending on libfdk-aac as well, but the binaries and the source code aren’t included in this PR anyway.

Reply:
I've removed libfdk-aac entirely